prow/labels/labels.go

```diff
@@ -18,24 +18,26 @@ package labels
 
 // labels for github plugins
 const (
-	LGTM            = "lgtm"
+	Approved        = "approved"
 	BlockedPaths    = "do-not-merge/blocked-paths"
-	LifecycleActive = "lifecycle/active"
-	LifecycleFrozen = "lifecycle/frozen"
-	LifecycleStale  = "lifecycle/stale"
-	LifecycleRotten = "lifecycle/rotten"
-	ClaYes          = "cncf-cla: yes"
+	Bug             = "kind/bug"
 	ClaNo           = "cncf-cla: no"
-	Approved        = "approved"
-	InvalidOwners   = "do-not-merge/invalid-owners-file"
-	CpUnapproved    = "do-not-merge/cherry-pick-not-approved"
+	ClaYes          = "cncf-cla: yes"
 	CpApproved      = "cherry-pick-approved"
-	WorkInProgress  = "do-not-merge/work-in-progress"
-	Hold            = "do-not-merge/hold"
-	Shrug           = "¯\\_(ツ)_/¯"
-	NeedsSig        = "needs-sig"
-	Bug             = "kind/bug"
-	Help            = "help wanted"
+	CpUnapproved    = "do-not-merge/cherry-pick-not-approved"
 	GoodFirstIssue  = "good first issue"
+	Help            = "help wanted"
+	Hold            = "do-not-merge/hold"
+	InvalidOwners   = "do-not-merge/invalid-owners-file"
+	LGTM            = "lgtm"
+	LifecycleActive = "lifecycle/active"
+	LifecycleFrozen = "lifecycle/frozen"
+	LifecycleRotten = "lifecycle/rotten"
+	LifecycleStale  = "lifecycle/stale"
+	NeedsOkToTest   = "needs-ok-to-test"
 	NeedsRebase     = "needs-rebase"
+	NeedsSig        = "needs-sig"
+	OkToTest        = "ok-to-test"
```

Comment:
Technically this is only used in the trigger package right now, so perhaps we don't need to export it?